### PR TITLE
SE-1917 fix flaky tests + add license

### PR DESCRIPTION
**Pull request by samuelallan72**
I don't think these tests were flaky actually, and I don't see any hacks in integration tests. Managed to fix a deprecation warning and add the license though.

**Test instructions**:

- verify that the license is added correctly (correct trove classifier in setup.py and correct LICENSE)
- verify that tests consistently pass for all support pythons

**Reviewer**:

- [x] @pomegranited

### CODE REVIEW

File: setup.py
Comment (pomegranited) [inline setup.py]: Please bump the version number -- we'll tag this once PR is merged too.

**Review by pomegranited — APPROVED** [review]
@swalladge -- One nit about bumping the version number, but this is ready to merge once that's done.

👍 

- [x] I tested this by checking the test logs for deprecation warnings and running the updated XBlock on my master devstack.
- [x] I read through the code, and checked the license additions.
- [x] <strike>I checked for accessibility issues</strike> No UI changes
- [x] Includes documentation
- [x] <strike>I made sure any change in configuration variables is reflected in the corresponding
    client's `configuration-secure` repository.</strike> N/A

File: setup.py
Comment (pomegranited) [reply setup.py]: @swalladge  Thanks for bumping the version here!  Could you create a tag for the merged commit, named to match the new `0.2.1` version?

Comment (samuelallan72) [reply setup.py]: Done :)

**Pull request merged**
